src/components/user/UpdateUserInformation.tsx

```diff
@@ -222,7 +223,7 @@ export default function UpdateUserInformation(props: { id: number }) {
             <Grid container spacing={3}>
               <Grid item xs={6}>
                 <LocalizationProvider dateAdapter={DateAdapter}>
-                  <FormikDropdown
+                  <FormikUIAutocomplete
```

Comment:
You should probably use simple select here the same as in SignUps.

If you want to use Autocomplete, there is a parameter for the AutoComplete component `autoComplete` which can be set to false

Reply:
Good point!